lib/sync.js

```diff
@@ -95,6 +97,11 @@ function Sync(src, target, opts, callback) {
     };
   }
 
+  if (opts.live) {
+    this.push.on('complete', self.pull.cancel.bind(self.pull));
+    this.pull.on('complete', self.push.cancel.bind(self.push));
+  }
+
```

Comment:
How is it possible for a user to call `pull.cancel()` when `sync()` only gives them the `Sync` object?